content/_data/changelogs/weekly.yml

```diff
@@ -392,6 +392,35 @@
         SSH CLI client authenticator 1.3. 
         Expose <code>PublicKeySignatureWriter</code> to plugins.
       pull: 2840
+- version: "2.56"
+  date: 2017-04-23
+  changes:
+    - type: rfe
+      message: >
+        Plugins may not have been expecting <code>InvalidPathException</code> since 2.55, so wrap then in <code>IOException</code> to restore previous behavior.
+      issue: 43531
+      pull: 2849
+    - type: bug
+      message: >
+        Remove links in stack traces to the <tt>stacktrace.jenkins-ci.org</tt> service that has been shut down.
+      issue: 42861
+      pull: 2811
+    - type: bug
+      message: >
+        If an exception is thrown while rendering an HTTP response, just log the stack trace on the server side, without trying to send an error page to the client.
```

Comment:
"If an exception is thrown for already sent response, just log the stack trace on the server side"